src/Swagger.ObjectModel/Primitive.cs

```diff
@@ -17,6 +17,7 @@ public class Primitive
             // Numbers
             { typeof(float), new Primitive("number", "float") },
             { typeof(double), new Primitive("number", "double") },
+            { typeof(decimal), new Primitive("number", "decimal") },
```

Comment:
I'm super new to swagger, docs seemed to think making up new types is okay, I don't actually know what the ramifications of this are, and if we need to add meaning to `decimal` elsewhere. I just needed it to handle `decimal?` without outputting it as just `Nullable`